tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -114,8 +117,13 @@ def type_comment(self, value):
         element.send_keys(value)
 
     def tap_comment(self):
-        self.marionette.find_element(*self._add_new_note_locator).tap()
+        element = self.marionette.find_element(*self._add_new_note_locator)
+        element.tap()
+        self.marionette.execute_script(
+            'arguments[0].scrollIntoView(true);', [element])
```

Comment:
This call is great! [Element.scrollIntoView()](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView) could be useful for the Graphics test in Settings. What do you think, @npark-mozilla ?

If so, how about moving that method into `Base.py`? Something like:
``` python
def scroll_to_element(element, should_be_at_top=True):
    self.marionette.execute_script(
        'arguments[0].scrollIntoView(arguments[1]);', [element, should_be_at_top])
```